source/Extensibility.Authentication/HostServices/IAuthCookieCreator.cs

```diff
@@ -1,13 +1,11 @@
 ﻿using System;
-using Nancy;
-using Nancy.Cookies;
 using Octopus.Node.Extensibility.Authentication.HostServices;
 
 namespace Octopus.Server.Extensibility.Authentication.HostServices
 {
     public interface IAuthCookieCreator
     {
-        //ObsoleteEx INancyCookie CreateAuthCookie(NancyContext context, Guid token, bool persistant);
-        INancyCookie[] CreateAuthCookies(Request request, Guid token, SessionExpiry expiry, bool? usingSecureConnection = null);
+        CookieOptions[] CreateAuthCookies(Guid token, SessionExpiry expiry, bool requestAppearsToBeHttps,
+            bool? usingSecureConnection = null);
```

Comment:
I think `usingSecureConnection` should turn into `forceSecure` or `forceSecureCookie` or something similar which matches up to the `CookieOptions.Secure`. This will give a better indication of the intent of the argument.

Reply:
Both fixed